lib/pf/UnifiedApi/Controller/Config/PortalModules.pm

```diff
@@ -92,6 +93,7 @@ our %TYPES_TO_FORMS = (
         URL
         ShowLocalAccount
         SSL_Inspection
+        Mfa
```

Comment:
I know its a bit late to the party but it would be better written as `MFA` instead of `Mfa` to stay consistent with the standards of our codebase